src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -125,7 +125,7 @@
      * Regular expression defined in the sam spec. Any alternative contig should match this regular expression 
      * TODO: replace the pattern with a constant : see https://github.com/samtools/htsjdk/pull/956/files
      */
-    private static final Pattern ALTERNATIVE_CONTIG_NAME_PATTERN = Pattern.compile("[0-9A-Za-z][0-9A-Za-z\\*\\+@\\|\\-]*");
+    private static final Pattern ALTERNATIVE_CONTIG_NAME_PATTERN = Pattern.compile("[0-9A-Za-z][0-9A-Za-z*+.@_|-]*");
```

Comment:
I think that there is no special alternative_contig_name pattern anymore, so you can use SAMSequenceRecord.LEGAL_RNAME_PATTERN